Sources/Compiler/IR/InstructionIndex.swift

```diff
@@ -8,28 +8,28 @@
 /// - SeeAlso: `InstructionIdentity`
 public struct InstructionIndex: Hashable {
 
-  /// The index of the function in which the instruction resides.
+  /// The function containing this index.
```

Comment:
same problem.  A thing doesn't contain its own indices, in general.

```suggestion
  /// The function in the module where this position is located.
```
or

```suggestion
  /// The function part of this position.
```

Or even,

```suggestion
  /// The function containing this position.
```

It's subtle, but this is the difference between the abstraction “position” and “index,” which indicates a datum.  The former works in that context, but the latter does not.